.travis.yml

```diff
@@ -5,6 +5,7 @@ rvm:
   - 2.6.0-preview2
   - 2.5.1
   - 2.4.4
+  - 2.3
```

Comment:
Can we use last exact version please, so builds are deterministic?
Trying to avoid different builds happening with no code changes, after new ruby versions released.